deel-3-webpagina/styles/styles.css

```diff
@@ -19,6 +19,7 @@ nav {
     display: flex;
     justify-content: space-between;
     align-items: baseline;
+    padding: 10px;
```

Comment:
add padding on the right side also so contact isn't pressed against the right side of the screen. same goes for the company name